source/contracts/reporting/Market.sol

```diff
@@ -322,10 +313,19 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
         return true;
     }
 
+    function doFeePayout(bool _toOwner, uint256 _amount) private returns (bool) {
+        if (_toOwner) {
+            getOwner().transfer(_amount);
```

Comment:
Speaking of not using `.transfer`... ;)  This should be `value(...)()`

Reply:
Hows that syntax work in this case exactly?

Reply:
If you mean `call.value(...)()` I'd actually argue against that since it's effectively `transfer` but more dangerous.